sections/futures/MarketInfoBox/MarketInfoBox.tsx

```diff
@@ -19,20 +25,82 @@ const MarketInfoBox: React.FC<MarketInfoBoxProps> = ({
 	buyingPower,
 	marginUsage,
 	isMarketClosed,
+	potentialTrade,
+	maxLeverageValue,
+	currencyKey,
 }) => {
+	const potentialTradeDetails = useGetFuturesPotentialTradeDetails(
+		currencyKey as CurrencyKey,
+		potentialTrade
+	);
+
+	const previewTrade = potentialTradeDetails.data ?? null;
+
+	const previewTradeData = React.useMemo(() => {
+		const size = wei(potentialTrade?.size || zeroBN);
+		const potentialAvailableMargin =
+			previewTrade?.margin?.sub(previewTrade?.minInitialMargin) ?? zeroBN;
+		const potentialBuyingPower = previewTrade?.margin?.mul(maxLeverageValue) ?? zeroBN;
+		const enumeratorMarginUsage = previewTrade?.margin?.sub(potentialAvailableMargin);
+		const potentialMarginUsage = enumeratorMarginUsage?.div(previewTrade?.margin)?.abs() ?? zeroBN;
```

Comment:
From our discussion: `margin usage = notional value / buying power`

I don't think "available margin" is relevant in this context at all. It should be something like:
`previewTrade.size.mul(previewTrade.price).div(potentialBuyingPower)`